src/components/input/math-input.js

```diff
@@ -187,7 +187,8 @@ class MathInput extends React.Component {
         window.addEventListener(
                 'orientationchange', this._clearKeypadBoundsCache);
 
-        window.document.addEventListener("focusout", this._keepInputFocus);
+        // TODO(nick): fix this so it doesn't break webapp
+        // window.document.addEventListener("focusout", this._keepInputFocus);
```

Comment:
I'll address this in a future PR- it deserves it's own work

Reply:
Is there a Jira ticket for this? In webapp we've been putting the ticket url in the todo comment for big work so it's easier to see the status of something when you find a note like this.